src/Core/Cache/ApcuCacheFactory.php

```diff
@@ -27,9 +27,13 @@ public function __construct($version = null)
      */
     public function create($service, array $params = array())
     {
+        $namespace = isset($params['namespace']) ? $params['namespace'] : '';
+        $defaultLifetime = isset($params['defaultLifetime']) ? $params['defaultLifetime'] : 0;
+        $directory = isset($params['directory']) ? $params['directory'] : null;
```

Comment:
I don't think we can rely on directly being passed, since this isn't a filesystem-backed cache and we'd be relying on users to set this manually, which is bad dev-experience.

I'd recommend adding BASE_PATH to the cache key instead.